cmd/neofs-cli/modules/container.go

```diff
@@ -290,7 +290,7 @@ var listContainerObjectsCmd = &cobra.Command{
 		objectIDs := res.IDList()
 
 		for i := range objectIDs {
-			cmd.Println(objectIDs[i])
+			cmd.Println(&objectIDs[i])
```

Comment:
You have suggested to use `.String()` in such cases: https://github.com/nspcc-dev/neofs-node/pull/1238#discussion_r826953610

Reply:
Fixed.